.github/workflows/update-sdk.yml

```diff
@@ -24,3 +24,4 @@ jobs:
       with:
         quality: 'daily'
         repo-token: ${{ secrets.GITHUB_TOKEN }}
+        branch-name: dev
```

Comment:
We actually don't want this, this controls the name of the PR branch. If the workflow auto-triggers on Monday, I think we should just delete the file from `main`.